### DISCUSSION TITLE
Is HaplotypeCaller compatible with Apple silicon?

### DISCUSSION THREAD

**Discussion by aiqc** [broadinstitute/gatk#8105] [Q&A]
Hello,

For variant calling purposes, I am considering upgrading to a many-cored Mac with the Apple silicon for parallelism, speed, and reduced power consumption.

However, I don't know if HaplotypeCaller supports the M1/M2 chips because of supposed Intel AVX acceleration:
https://www.biostars.org/p/478429/#495387

Has anyone gotten it working? Are there other issues that I should be aware of?

Thank you

**Comment by aiqc** [comment]
Cross-posted after waiting 1.5 weeks: https://gatk.broadinstitute.org/hc/en-us/community/posts/10937066510747-Do-GATK-tools-support-Apple-silicon-

**Comment by lbergelson** [comment]
@aiqc HaplotypeCaller technically works if you disable the accelerated pair hmm but it's unusably slow for human genome scale stuff.  M1/M2 don't provide translations for the AVX instruction set and we don't have anyone who is current able to adapt the acceleration for arm.   If you're running small data you could run it.

**Reply by aiqc** [reply]
Thank you for the insight @lbergelson. Any idea on how much acceleration is happening: is it 10x or more like 2x?
My use case is WES, so 50x smaller than WGS.

**Comment by lbergelson** [comment]
The forum is definitely the best place to go for questions like this in the future.
